app/src/main/res/values/styles.xml

```diff
@@ -385,8 +385,8 @@
     </style>
 
     <style name="OverflowMenuItem" parent="ListItem">
-        <item name="android:paddingTop">13dp</item>
-        <item name="android:paddingBottom">13dp</item>
+        <item name="android:paddingTop">10dp</item>
+        <item name="android:paddingBottom">10dp</item>
```

Comment:
Have you checked with the designer that this is a change across the app? We have used the `OverflowMenuItem` style in other places.

Reply:
I have not confirmed this with the designer yet. Thanks for flagging that, I will reach out to confirm if this is a global change across the app.